tests/test_annoy_console.py

```diff
@@ -0,0 +1,33 @@
+# Unit this for annoy
+# Import slogpy from slog
+from slogpy.slog import Slog as slog  # noqa: N813, E402
+
+"""This module contains tests for the Slog class, specifically the annoying method,
+which is responsible for slog annoying messages.
+"""
+
+# (note to self) make sure the count is only once......
+
+def test_annoy_unique_message(capsys, msg="This is a annoy test"):  # noqa: D103
+    """Timestamp checking method
```

Comment:
I don't see any timestamp being checked and that is not the name of the function, nor does it actually appear to be checking a timestamp.